internal/provisioner/objects/deployment/deployment_test.go

```diff
@@ -152,7 +152,8 @@ func TestDesiredDeployment(t *testing.T) {
 		"key": "value",
 	}
 
-	// Use non-default container ports to test that --envoy-service-http(s)-port
+	// Use non-default container ports to test that
```

Comment:
Let's revert the no-op changes to deployment.go and deployment_test.go, for clarity in the commit.